azext_iot/central/services/device.py

```diff
@@ -294,5 +295,75 @@ def get_component_command_history(
     )
     headers = _utility.get_headers(token, cmd)
 
-    response = requests.get(url, headers=headers)
+    response = requests.get(url, headers=headers, verify=not should_disable_connection_verify())
+    return _utility.try_extract_result(response)
+
+
+def run_manual_failover(
+    cmd,
+    app_id: str,
+    device_id: str,
+    ttl_minutes: int = None,
+    token: str = None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+):
+    """
+    Execute a manual failover of device across multiple IoT Hubs to validate device firmware's ability to reconnect using DPS to a different IoT Hub.
+
+    Args:
+        cmd: command passed into az
+        app_id: id of an app (used for forming request URL)
+        device_id: unique case-sensitive device id
+        ttl_minutes: (OPTIONAL) An optional value to specify the expiration time of this manual failover test before the device moves back to it's original IoT Hub. 
+            This has a default value of 30 minutes, but can optionally be any positive integer between 1 and 30. 
+        token: (OPTIONAL) authorization token to fetch device details from IoTC.
+            MUST INCLUDE type (e.g. 'SharedAccessToken ...', 'Bearer ...')
+        central_dns_suffix:(OPTIONAL) {centralDnsSuffixInPath} as found in docs
+
+    Returns:
+        result (currently a 200)
+    """
+
+    url = "https://{}.{}/{}/{}/manual-failover".format(
+        app_id, central_dns_suffix, "system/iothub/devices", device_id
+    )
+    headers = _utility.get_headers(token, cmd)
+    json = {}
+    if ttl_minutes :
```

Comment:
is TTL 0 valid? because if so this won't get populated on 0 case

Reply:
TTLL is optional parameter for backend, if not specified than default value will be assigned by backend. default can be updated over time in backend